tests/functional/builtins/codegen/test_extract32.py

```diff
@@ -34,13 +34,15 @@ def extrakt32_storage(index: uint256, inp: Bytes[100]) -> bytes32:
     )
 
     for S, i in test_cases:
-        expected_result = S[i : i + 32] if 0 <= i <= len(S) - 32 else None
-        if expected_result is None:
-            assert_tx_failed(lambda: c.extrakt32(S, i))
-        else:
-            assert c.extrakt32(S, i) == expected_result
-            assert c.extrakt32_mem(S, i) == expected_result
-            assert c.extrakt32_storage(i, S) == expected_result
+        if i < 0 or i > len(S) - 32:
```

Comment:
i think i prefer the original comparison form
```suggestion
        if 0 <= i <= len(S) - 32:
```